app/services/instrument_offline_reservation_canceler.rb

```diff
@@ -0,0 +1,44 @@
+class InstrumentOfflineReservationCanceler
+
+  def cancel!
+    reservations_to_cancel.each do |reservation|
+      reservation.transaction do
+        cancel_reservation(reservation)
+        Notifier.offline_cancellation_notification(reservation)
+      end
+    end
+  end
+
+  private
+
+  def admin
```

Comment:
Could this be called something like `admin_user` or `admin_dummy_user` so it's a little clearer how it's being used?